world-builder/app/models/user.rb

```diff
@@ -3,6 +3,5 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-    has_many :characters
```

Comment:
This should be changed to 'has_many :user_campaigns' and 'has_many :user_campaigns' in campaign.rb should be changed too

Reply:
sounds good ill change that

Reply:
added